www/views.py

```diff
@@ -42,7 +42,7 @@ def prepare_message(request, project, m, detailed):
     name, addr = m.sender
     m.sender_full_name = "%s <%s>" % (name, addr)
     m.sender_display_name = name or addr
-    m.age = m.get_age()
+    m.age = m.date
```

Comment:
This can go away too. Age and date are semantically different, and the only usage is filtered in the template.

Reply:
Makes sense, updated!